UnitySDK/Assets/ML-Agents/Scripts/InferenceBrain/ModelRunner.cs

```diff
@@ -66,8 +67,10 @@ public class ModelRunner : IBatchedDecisionMaker
 
             m_InferenceInputs = BarracudaModelParamLoader.GetInputTensors(barracudaModel);
             m_OutputNames = BarracudaModelParamLoader.GetOutputNames(barracudaModel);
-            m_TensorGenerator = new TensorGenerator(brainParameters, seed, m_TensorAllocator, barracudaModel);
-            m_TensorApplier = new TensorApplier(brainParameters, seed, m_TensorAllocator, barracudaModel);
+            m_TensorGenerator = new TensorGenerator(
+                brainParameters, seed, m_TensorAllocator, ref m_Memories, barracudaModel);
```

Comment:
Don't think we need `ref` here now, right?

Reply:
➕